core-feature-pack/src/main/resources/content/domain/configuration/host-master.xml

```diff
@@ -5,6 +5,9 @@
    and does not itself directly control any servers.
 -->
 <host name="master" xmlns="urn:jboss:domain:3.0">
+    <extensions>
+        <extension module="org.jboss.as.jmx"/>
```

Comment:
please don't use jmx testing  / default configs as i think we still want to move it out of core repo.